app/helpers/miq_action_helper.rb

```diff
@@ -123,6 +123,7 @@ def miq_summary_action_type(record, alert_guids, cats)
       rows.push({:cells => {:label => _("Categories"), :value => cats}})
     end
 
+    data[:rows] = rows
```

Comment:
I think this fix is in https://github.com/ManageIQ/manageiq-ui-classic/pull/9086